util/src/Pos/Util/Log/LoggerConfig.hs

```diff
@@ -171,7 +178,7 @@ mkUniq handlers = mkUniq' handlers []
             any (\lh' -> _lhName lh == _lhName lh') lhs
         mkUniq' [] acc = acc
         mkUniq' (lh:lhs) acc | lh `containedIn` acc = mkUniq' lhs acc
-                             | otherwise            = mkUniq' lhs (lh:acc)
```

Comment:
I actually don't want the order to change.
You don't like `CONS`?

Reply:
Well, in this scenario 

```hs
let mergedConfig = loggerConfig1 <> loggerConfig2
```

where both `loggerConfig1` and `loggerConfig2` define a `LogHandler` with a same name (let's say, "console"), If one had to prevail, I'd expect the one from `loggerConfig2` to prevail upon `loggerConfig1`. 
However, using pre-pending handlers to the list like this will give prevalence to the left-most config. 
 
In other words, `Semigroup` is supposed to be _right-associative_ on `<>`, so objects should be grouped from right to left. Hence why we should append things and not prepend them.